src/cpu/amd/pi/Makefile.inc

```diff
@@ -30,6 +30,14 @@ romstage-y += heapmanager.c
 ramstage-y += heapmanager.c
 ramstage-y += amd_late_init.c
 
+romstage-$(CONFIG_CPU_AMD_FAM16H_MICROCODE) += microcode_fam16h.c
+ramstage-$(CONFIG_CPU_AMD_FAM16H_MICROCODE) += microcode_fam16h.c
+
+# Microcode for Family 16h
+cbfs-files-$(CONFIG_CPU_AMD_FAM16H_MICROCODE) += microcode_amd.bin
```

Comment:
@miczyg1 I think it would be much better to give user option to add microcode binary from filesystem, then add binary to our tree.